### PR TITLE
[Messaging-extensions][Add-sections][1439845]

### PR DESCRIPTION
**Pull request by Rajeshwari-v**
Added missing sections. Please review.

### CODE REVIEW

**Review by KirtiPereira — CHANGES_REQUESTED** [review]
Thorough self-review required.

File: msteams-platform/messaging-extensions/how-to/action-commands/create-task-module.md [hunk 1]
```diff
@@ -463,9 +463,71 @@ class TeamsMessagingExtensionsActionPreview extends TeamsActivityHandler {
...
+```json
+{
+  "type": "AdaptiveCard",
+  "body": [
+    {
+      "type": "TextBlock",
+      "text": "Looks like you haven't used Disco in this team/chat"
+    }
+  ],
+  "actions": [
```
Comment (Wajeed-msft) [inline msteams-platform/messaging-extensions/how-to/action-commands/create-task-module.md:483]: Could you please add C# version as well, please work with Nikitha?  Here is the link to the sample code: https://github.com/microsoft/BotBuilder-Samples/blob/main/samples/csharp_dotnetcore/51.teams-messaging-extensions-action

**Review by Nikitha-MSFT — APPROVED** [review]
Reviewed and approved

**Comment by Rajeshwari-v** [comment]
@surbhigupta12 , @KirtiPereira - Closing this PR as create-messaging-extension.md file details will be re-structured and updated in respective files. 
Information updated in create-task-module.md file for this PR is combined with PR #2664.

**Pull request closed without merging**